packages/data-context/src/DataContext.ts

```diff
@@ -58,7 +58,11 @@ export class DataContext extends DataContextShell {
     ]
 
     if (this.config.launchArgs.projectRoot) {
-      toAwait.push(this.actions.project.setActiveProject(this.config.launchArgs.projectRoot))
+      await this.actions.project.setActiveProject(this.config.launchArgs.projectRoot)
+
+      if (this.coreData.app.activeProject?.preferences) {
+        toAwait.push(this.actions.project.launchProjectWithoutLaunchpad())
```

Comment:
I wonder if we should name this something more generic, like `launchElectron`? Eg, we might one day not have a launchpad, but it's likely we will always have some kind of electron app running.